x/currencies/client/cli/tx.go

```diff
@@ -1,50 +1,123 @@
-// Transaction commands for currencies CLI implementation.
 package cli
 
 import (
-	"bufio"
-	"fmt"
 	"os"
 
-	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
-	txBldrCtx "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	govCli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
 	"github.com/spf13/cobra"
 
-	"github.com/dfinance/dnode/x/currencies/msgs"
+	"github.com/dfinance/dnode/helpers"
+	"github.com/dfinance/dnode/x/currencies/internal/types"
 )
 
-// Destroy currency.
-func PostDestroyCurrency(cdc *codec.Codec) *cobra.Command {
-	return &cobra.Command{
-		Use:   "destroy-currency [chainID] [symbol] [amount] [recipient]",
-		Short: "destroy issued currency",
-		Args:  cobra.ExactArgs(4),
+// PostWithdrawCurrency returns tx command which post a new withdraw request.
+func PostWithdrawCurrency(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "withdraw-currency [denom] [amount] [pegZoneSpender] [pegZoneChainID]",
+		Short:   "Withdraw issued currency, reducing spender balance",
```

Comment:
"Withdraw issued currency from dfinance chain to pegzone chain..."

Reply:
Changed
https://github.com/dfinance/dnode/pull/152/commits/0ac2565d2234e713d801f65f9bcea740d4f95883